client/src/components/pages/RobotOverview/RobotOverview.test.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable func-names */
+/* eslint-disable object-shorthand */
+/* eslint-disable no-undef */
+import React from 'react';
+import {act, render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { BrowserRouter } from 'react-router-dom';
+import RobotOverview from './RobotOverview';
+import '@testing-library/jest-dom'
+
+const USER_ID = '80625d115100a2ee8d8e695b';
+const NEW_ROBOT_NAME = 'New Robot';
+
+window.matchMedia = window.matchMedia || function() {
```

Comment:
Can you explain to me, what do the` || `do here? I haven't seen this yet 😄

Reply:
Honestly no idea. I had an error saying window.matchMedia was undefined. Upon googling I found multiple approaches to solve it, this worked for me.
My understanding is that it works like „if window.matchMedia id defined use it, otherwise use the following function“